jetty-server/src/test/java/org/eclipse/jetty/server/HttpServerTestBase.java

```diff
@@ -209,8 +209,16 @@ public void testFullMethod() throws Exception
             byte[] buffer = new byte[64 * 1024];
             Arrays.fill(buffer, (byte)'A');
 
-            os.write(buffer);
-            os.flush();
+            try
+            {
+                os.write(buffer);
+                os.flush();
+            }
+            catch (Exception e)
+            {
+                if (LOG.isDebugEnabled())
+                    LOG.warn("write exception", e);
```

Comment:
is debug, then warn?

Reply:
Should be `LOG.debug()`, not `LOG.warn()`. It's just a test case, no need to be fancy and we will be consistent with the rest of the code.